modules/mono/editor/GodotTools/GodotTools/Build/MsBuildFinder.cs

```diff
@@ -161,8 +161,21 @@ private static string FindMsBuildToolsPathOnWindows()
 
             // Try to find 15.0 with vswhere
 
-            string vsWherePath = Environment.GetEnvironmentVariable(Internal.GodotIs32Bits() ? "ProgramFiles" : "ProgramFiles(x86)");
-            vsWherePath += "\\Microsoft Visual Studio\\Installer\\vswhere.exe";
+            var envNames = Internal.GodotIs32Bits() ? new[] { "ProgramFiles","ProgramFiles(x86)" } : new[] { "ProgramFiles(x86)","ProgramFiles" };
```

Comment:
There's no `ProgramFiles(x86)` on 32-bit Windows AFAIK:
```suggestion
            var envNames = Internal.GodotIs32Bits() ? new[] { "ProgramFiles" } : new[] { "ProgramFiles(x86)","ProgramFiles" };
```